src/authDataManager/twitter.js

```diff
@@ -12,7 +12,7 @@ function validateAuthData(authData, options) {
   client.auth_token_secret = authData.auth_token_secret;
   
   return client.get("/1.1/account/verify_credentials.json").then((data) => {
-    if (data && data.id == authData.id) {
+    if (data && data.id_str == authData.user_id) {
```

Comment:
The response from twitter does not include an attribute id (data.id). This is a possible way to solve the problem as the response does include the id_str which is the account id (which is included when requesting a token).